### ISSUE TITLE
Vosk support

### ISSUE THREAD

**Issue by OSS542** [synesthesiam/voice2json#47]
Is there any chance of adding support for Vosk to voice2json ?

**Comment by synesthesiam** [comment]
Yes! I have this partially working already. For open transcription, it works just fine. But for pre-trained voice command recognition, I've only found the ability to restrict the vocabulary. I may have to dig deeper to get grammar-based recognition working.

Were you planning to use Vosk for open transcription or restricted voice commands?

**Comment by OSS542** [comment]
At present I plan on using Vosk for open transcription, as it is better suited to that at present.  I use PocketSphinx for restricted vocabulary, as it is considerably faster at this, and the vocabulary is easy to modify.
